webapp/src/actions.ts

```diff
@@ -606,3 +607,17 @@ export const lowerHand = async (callID: string, sessionID: string) => {
         },
     );
 };
+
+export const hostRemove = async (callID?: string, sessionID?: string) => {
+    if (!callID || !sessionID) {
+        return {};
+    }
+
+    return RestClient.fetch(
+        `${getPluginPath()}/calls/${callID}/host/remove`,
```

Comment:
Random thought since we've been battling with issues. Would this work properly aginst a server running under a subpath? I see we are handling that case in `getPluginPath()` but now we are also setting the path to the client itself on initialization. Just want to double check the two things are not conflicting.

Reply:
Yes it works, just tested.